Your pull request does not meet our code style rules. Pylint summary:
```
************* Module accumulator
testsuite/python/accumulator.py:24:0: W0611: Unused import sys (unused-import)
```

You can generate these warnings with `maintainer/CI/fix_style.sh`. This is the same command that I have executed to generate the log above.

--- a/doc/sphinx/analysis.rst
+++ b/doc/sphinx/analysis.rst
@@ -739,8 +739,8 @@ In order to calculate the running mean and variance of an observable,
         obs=position_observable, delta_N=2)
     system.auto_update_accumulators.add(accumulator)
     system.integrator.run(10)
-    print(accumulator.get_mean())
-    print(accumulator.get_variance())
+    print(accumulator.mean())
+    print(accumulator.variance())
 
 In the example above the automatic update of the accumulator is used. However,
 it's also possible to manually update the accumulator by calling

--- a/doc/tutorials/charged_system/charged_system-1.ipynb
+++ b/doc/tutorials/charged_system/charged_system-1.ipynb
@@ -597,7 +597,7 @@
     "**Hints:**\n",
     "* Don't forget to clear the system before setting up the system with a new set of parameters\n",
     "* Don't forget to ``tune()`` the ``p3m`` instance after each change of parameters. If we reuse the p3m that was tuned before, likely the desired accuracy will not be achieved. \n",
-    "* Extract the radial density profile from the accumulator via ``.get_mean()``"
+    "* Extract the radial density profile from the accumulator via ``.mean()``"
    ]
   },
   {
@@ -622,7 +622,7 @@
     "    print('starting simulation')\n",
     "    integrate_system(system, N_SAMPLES * STEPS_PER_SAMPLE)\n",
     "\n",
-    "    run['histogram'] = radial_profile_accs[COUNTERION_TYPE].get_mean()\n",
+    "    run['histogram'] = radial_profile_accs[COUNTERION_TYPE].mean()\n",
     "    print('simulation for parameters {} done\\n'.format(run['params']))\n",
     "```"
    ]
@@ -862,7 +862,7 @@
     "rs = bin_edges[1:, 0, 0, 0]\n",
     "cum_hists = {}\n",
     "for ion_type in all_ion_types:\n",
-    "    hist = radial_profile_accs[ion_type].get_mean()\n",
+    "    hist = radial_profile_accs[ion_type].mean()\n",
     "    hist = hist[:, 0, 0] * rs\n",
     "    cum_hist = np.cumsum(hist)\n",
     "    cum_hist /= cum_hist[-1]\n",

--- a/doc/tutorials/lennard_jones/lennard_jones.ipynb
+++ b/doc/tutorials/lennard_jones/lennard_jones.ipynb
@@ -956,7 +956,7 @@
    },
    "source": [
     "```python\n",
-    "rdf = rdf_acc.get_mean()\n",
+    "rdf = rdf_acc.mean()\n",
     "rs = rdf_obs.bin_centers()\n",
     "```"
    ]

--- a/samples/drude_bmimpf6.py
+++ b/samples/drude_bmimpf6.py
@@ -402,9 +402,9 @@ else:
 
     file_traj.close()
 
-    rdf_00 = acc_00.get_mean()
-    rdf_11 = acc_11.get_mean()
-    rdf_01 = acc_01.get_mean()
+    rdf_00 = acc_00.mean()
+    rdf_11 = acc_11.mean()
+    rdf_01 = acc_01.mean()
     r = obs_01.bin_centers()
     np.savetxt(args.path + "rdf.dat", np.c_[r, rdf_00, rdf_11, rdf_01])
     print("\n-->Done")

--- a/samples/electrophoresis.py
+++ b/samples/electrophoresis.py
@@ -214,10 +214,10 @@ logging.info("persistence length (python analysis): {}".format(
 
 def persistence_length_obs(
         acc_bond_length, acc_persistence_angles, exponential):
-    bond_lengths_obs = np.array(acc_bond_length.get_mean())
+    bond_lengths_obs = np.array(acc_bond_length.mean())
     sampling_positions_obs = np.insert(
         np.cumsum(bond_lengths_obs)[:-1], 0, 0.0)
-    cos_thetas_obs = np.array(acc_persistence_angles.get_mean())
+    cos_thetas_obs = np.array(acc_persistence_angles.mean())
     cos_thetas_obs = np.insert(cos_thetas_obs, 0, 1.0)
 
     opt_obs, _ = scipy.optimize.curve_fit(

--- a/samples/lb_profile.py
+++ b/samples/lb_profile.py
@@ -70,7 +70,7 @@ accumulator = espressomd.accumulators.MeanVarianceCalculator(obs=fluid_obs)
 system.auto_update_accumulators.add(accumulator)
 system.integrator.run(n_steps)
 
-lb_fluid_profile = accumulator.get_mean()
+lb_fluid_profile = accumulator.mean()
 
 
 def poiseuille_flow(r, R, ext_force_density):

--- a/src/core/accumulators/MeanVarianceCalculator.cpp
+++ b/src/core/accumulators/MeanVarianceCalculator.cpp
@@ -33,9 +33,7 @@
 namespace Accumulators {
 void MeanVarianceCalculator::update() { m_acc(m_obs->operator()()); }
 
-std::vector<double> MeanVarianceCalculator::mean() {
-  return m_acc.mean();
-}
+std::vector<double> MeanVarianceCalculator::mean() { return m_acc.mean(); }
 
 std::vector<double> MeanVarianceCalculator::variance() {
   return m_acc.variance();

--- a/src/core/reaction_ensemble.cpp
+++ b/src/core/reaction_ensemble.cpp
@@ -1611,14 +1611,13 @@ WidomInsertion::measure_excess_chemical_potential(int reaction_id) {
   current_reaction.accumulator_exponentials(exponential);
 
   std::pair<double, double> result = std::make_pair(
-      -temperature *
-          log(current_reaction.accumulator_exponentials.mean()[0]),
+      -temperature * log(current_reaction.accumulator_exponentials.mean()[0]),
       std::abs(-temperature /
                current_reaction.accumulator_exponentials.mean()[0] *
                current_reaction.accumulator_exponentials
                    .std_error()[0])); // excess chemical potential; error
-                                          // excess chemical potential,
-                                          // determined via error propagation
+                                      // excess chemical potential,
+                                      // determined via error propagation
   return result;
 }
 

--- a/src/python/espressomd/accumulators.py
+++ b/src/python/espressomd/accumulators.py
@@ -43,20 +43,20 @@ class MeanVarianceCalculator(ScriptInterfaceHelper):
     )
     _so_creation_policy = "LOCAL"
 
-    def get_mean(self):
+    def mean(self):
         """
         Returns the samples mean values of the respective observable with
         which the accumulator was initialized.
         """
         return np.array(self.call_method("mean")).reshape(self.shape())
 
-    def get_variance(self):
+    def variance(self):
         """
         Returns the samples variance for the observable.
         """
         return np.array(self.call_method("variance")).reshape(self.shape())
 
-    def get_std_error(self):
+    def std_error(self):
         """
         Returns the standard error calculated from the samples variance for the observable by
         assuming uncorrelated samples.

--- a/src/script_interface/accumulators/MeanVarianceCalculator.hpp
+++ b/src/script_interface/accumulators/MeanVarianceCalculator.hpp
@@ -61,10 +61,14 @@ public:
 
   Variant do_call_method(std::string const &method,
                          VariantMap const &parameters) override {
-    if (method == "update") mean_variance_calculator()->update();
-    if (method == "mean") return mean_variance_calculator()->mean();
-    if (method == "variance") return mean_variance_calculator()->variance();
-    if (method == "std_error") return mean_variance_calculator()->std_error();
+    if (method == "update")
+      mean_variance_calculator()->update();
+    if (method == "mean")
+      return mean_variance_calculator()->mean();
+    if (method == "variance")
+      return mean_variance_calculator()->variance();
+    if (method == "std_error")
+      return mean_variance_calculator()->std_error();
     return AccumulatorBase::call_method(method, parameters);
   }
 

--- a/src/utils/include/utils/Accumulator.hpp
+++ b/src/utils/include/utils/Accumulator.hpp
@@ -102,20 +102,18 @@ inline std::vector<double> Accumulator::variance() const {
     res = std::vector<double>(m_acc_data.size(),
                               std::numeric_limits<double>::max());
   } else {
-    std::transform(
-        m_acc_data.begin(), m_acc_data.end(), std::back_inserter(res),
-        [this](const AccumulatorData<double> &acc_data) {
-          return acc_data.m /
-                 (static_cast<double>(m_n) -
-                  1);
-        });
+    std::transform(m_acc_data.begin(), m_acc_data.end(),
+                   std::back_inserter(res),
+                   [this](const AccumulatorData<double> &acc_data) {
+                     return acc_data.m / (static_cast<double>(m_n) - 1);
+                   });
   }
   return res;
 }
 
 /**
  * Returns the standard error of the mean assuming uncorrelated samples.
-*/
+ */
 inline std::vector<double> Accumulator::std_error() const {
   auto const var = variance();
   std::vector<double> err(var.size());

--- a/src/utils/tests/accumulator.cpp
+++ b/src/utils/tests/accumulator.cpp
@@ -35,10 +35,9 @@ BOOST_AUTO_TEST_CASE(accumulator) {
   BOOST_CHECK(acc.variance() ==
               std::vector<double>(4, std::numeric_limits<double>::max()));
   acc(test_data2);
+  BOOST_CHECK((acc.mean() == std::vector<double>{{0.75, 2.25, 2.75, 3.75}}));
   BOOST_CHECK(
-      (acc.mean() == std::vector<double>{{0.75, 2.25, 2.75, 3.75}}));
-  BOOST_CHECK((acc.variance() ==
-               std::vector<double>{{1.125, 3.125, 1.125, 1.125}}));
+      (acc.variance() == std::vector<double>{{1.125, 3.125, 1.125, 1.125}}));
   BOOST_CHECK(
       (acc.std_error() == std::vector<double>{{0.75, 1.25, 0.75, 0.75}}));
 }

--- a/testsuite/python/accumulator.py
+++ b/testsuite/python/accumulator.py
@@ -21,9 +21,10 @@
 Testmodule for the observable accumulator.
 
 """
-import sys
 import unittest as ut
+
 import numpy as np
+
 import espressomd
 import espressomd.observables
 import espressomd.accumulators
@@ -63,14 +64,14 @@ class AccumulatorTest(ut.TestCase):
             self.system.integrator.run(1)
         self.assertEqual(self.pos_obs, self.pos_obs_acc.get_params()['obs'])
         np.testing.assert_allclose(
-            self.pos_obs_acc.get_mean(),
+            self.pos_obs_acc.mean(),
             np.mean(self.positions, axis=0), atol=1e-4)
         variance = np.var(self.positions, axis=0, ddof=1)
         np.testing.assert_allclose(
-            self.pos_obs_acc.get_variance(),
+            self.pos_obs_acc.variance(),
             variance, atol=1e-4)
         np.testing.assert_allclose(
-            self.pos_obs_acc.get_std_error(),
+            self.pos_obs_acc.std_error(),
             np.sqrt(variance / len(self.positions)), atol=1e-4)
 
 

--- a/testsuite/python/lb_poiseuille_cylinder.py
+++ b/testsuite/python/lb_poiseuille_cylinder.py
@@ -159,11 +159,7 @@ class LBPoiseuilleCommon:
         self.prepare_obs()
         # gather some statistics for the observable accumulator
         self.system.integrator.run(1)
-        obs_result = np.array(
-            self.accumulator.get_mean()).reshape(OBS_PARAMS['n_r_bins'],
-                                                 OBS_PARAMS['n_phi_bins'],
-                                                 OBS_PARAMS['n_z_bins'],
-                                                 3)
+        obs_result = self.accumulator.mean()
         x = np.linspace(
             OBS_PARAMS['min_r'],
             OBS_PARAMS['max_r'],

--- a/testsuite/python/test_checkpoint.py
+++ b/testsuite/python/test_checkpoint.py
@@ -292,13 +292,13 @@ class CheckpointTest(ut.TestCase):
 
     def test_mean_variance_calculator(self):
         np.testing.assert_array_equal(
-            acc_mean_variance.get_mean(),
+            acc_mean_variance.mean(),
             np.array([[1.0, 1.5, 2.0], [1.0, 1.0, 2.0]]))
         np.testing.assert_array_equal(
-            acc_mean_variance.get_variance(),
+            acc_mean_variance.variance(),
             np.array([[0., 0.5, 2.], [0., 0., 0.]]))
         np.testing.assert_array_equal(
-            system.auto_update_accumulators[0].get_variance(),
+            system.auto_update_accumulators[0].variance(),
             np.array([[0., 0.5, 2.], [0., 0., 0.]]))
 
     def test_time_series(self):

